.github/workflows/ci.yml

```diff
@@ -113,16 +113,79 @@ jobs:
             -   name: Checkout
                 uses: actions/checkout@v3
 
+            -   name: Setup PHP to install dependencies
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 5.5
+                    ini-values: error_reporting=-1, display_errors=On, zend.assertions=1
+                    coverage: none
+                    # Autoload files generated with Composer 2.3+ are not compatible with PHP < 7.0.
+                    tools: composer:2.2
+
+            -   name: Install Composer dependencies
+                uses: ramsey/composer-install@v2
+                with:
+                    composer-options: "--no-dev --optimize-autoloader"
+
+            -   name: Setup PHP to generate the PHAR
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 8.1
+                    extensions: exif, phar, openssl, sodium
```

Comment:
why we need exif and sodium?

Reply:
Sodium is a requirement for Box 4. I'd need to check on why the `exif`, may be a copy/paste artifact.

Refs:
* https://github.com/box-project/box/blob/79136052a6d665e5ffa9f24f45cd9a272c170ca8/composer.json#L44-L46
* https://github.com/box-project/box/blob/master/UPGRADE.md#from-3x-to-4x

Reply:
Removed `exif` now.